mettle/worker/preflight.go

```diff
@@ -0,0 +1,85 @@
+package worker
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/bazelbuild/remote-apis-sdks/go/pkg/digest"
+	"github.com/bazelbuild/remote-apis-sdks/go/pkg/uploadinfo"
+	pb "github.com/bazelbuild/remote-apis/build/bazel/remote/execution/v2"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/durationpb"
+)
+
+// runPreflightAction runs a simple known action to ensure we can process them correctly.
+func (w *worker) runPreflightAction() error {
+	log.Notice("Preparing pre-flight action...")
+	const fileContents = "thirty-five ham and cheese sandwiches\n"
```

Comment:
This seems like a lot of sandwiches! It's probably fine but worth keeping an eye on it whilst this gets rolled out.

Reply:
Yup agreed. We should be able to roll  this out to just our staging environment first so it'll only be a few that get it.

Reply:
Right but if we're only rolling it out to staging, won't there be fewer consumers? Perhaps we should take this straight to prod to avoid wasted sandwiches.